sql/structured.go

```diff
@@ -242,6 +242,16 @@ func (desc *TableDescriptor) FindColumnByID(id ColumnID) (*ColumnDescriptor, err
 	return nil, fmt.Errorf("column-id \"%d\" does not exist", id)
 }
 
+// GetIndexByName finds the index with specified name.
+func (desc *TableDescriptor) GetIndexByName(name string) *IndexDescriptor {
+	for i, idx := range desc.Indexes {
+		if idx.Name == name {
```

Comment:
This comparison should use `equalName(idx.Name, name)`. Since you're touching this code, `FindColumnByName` should be changed similarly.